packages/v4/patternfly-docs/pages/home.js

```diff
@@ -10,269 +24,279 @@ import {
   Title,
   TextContent,
   Text,
-  TextVariants
-} from '@patternfly/react-core';
-import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
-import GithubIcon from '@patternfly/react-icons/dist/esm/icons/github-icon';
-import orb from '../images/pf4-logo.svg';
-import accessible from '../images/accessible.svg';
-import modular from '../images/modular.svg';
-import scale from '../images/scale.svg';
-import consistent from '../images/consistent.svg';
-import open from '../images/open.svg';
-import { AutoLinkHeader } from '@patternfly/documentation-framework/components/autoLinkHeader/autoLinkHeader';
-import './home.css';
+  TextVariants,
+} from "@patternfly/react-core";
+import ArrowRightIcon from "@patternfly/react-icons/dist/esm/icons/arrow-right-icon";
+import GithubIcon from "@patternfly/react-icons/dist/esm/icons/github-icon";
+import ExternalLinkAltIcon from "@patternfly/react-icons/dist/esm/icons/external-link-alt-icon";
+import lightshowcase from "../images/light-showcase.svg";
+import pfLogo from "../images/PF5-IconLogo-color.svg";
+import {AutoLinkHeader} from "@patternfly/documentation-framework/components/autoLinkHeader/autoLinkHeader";
+import "./home.css";
+import featuredPostsData from "./featured-posts-data.json";
 
-const aboutPatternFly = `Hello, we're PatternFly.`;
-const patternFlyPrinciples = 'Flyer fundamentals';
+const patternFlyPrinciples = "Flyer fundamentals";
+
+const cardData = [
+  {
+    title: "Read our design tokens guide",
+    body: "We swapped global CSS variables for design tokens for v5",
+    link: "",
+    layout: "withSubtitle",
+  },
+  {
+    title: "Migrate to Patternfly v5",
+    body: "We can help you plan a migration strategy and upgrade to v5",
+    link: "",
+    layout: "withSubtitle",
+  },
+  {
+    title: "Read our latest release notes",
+    body: "We outline code changes and design updates with every PatternFly release",
+    link: "",
+    layout: "withSubtitle",
+  },
+  {
+    title: "Explore our roadmaps",
+    body: "We always share what we’re working on now and what we have planned next",
+    link: "",
+    layout: "withSubtitle",
+    hasExtLinkIcon: true,
+  },
```

Comment:
```suggestion
 
```
I'm wondering if linking to the roadmaps is still helpful. Do we still want to do this? I believe we stopped pointing to them on the community page, so I'm wondering if we're moving away from them. @mcarrano

Reply:
@andrew-ronaldson and @srambach would be great to get your input on this.

Reply:
I'm reluctant to have the roadmap highlighted until we iron out the next two quarters. The tentative Q3 plan should be ready before the site goes live so we should be okay but curious what @srambach thinks?

I think it's not a bad idea to have it out there for transparency and it promotes the new quarterly release schedule.

Reply:
I definitely think it should be included, but I'm not entirely positive what to point to at the moment.